app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -13,10 +14,49 @@
   }
 }
 
-.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item {
+.gem-c-breadcrumbs--inverse.govuk-breadcrumbs--collapse-on-mobile .govuk-breadcrumbs__list-item {
   color: govuk-colour("white");
 }
 
-.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
+.gem-c-breadcrumbs--inverse.govuk-breadcrumbs--collapse-on-mobile .govuk-breadcrumbs__list-item:after {
   border-color: govuk-colour("white");
 }
+
+.govuk-breadcrumbs--collapse-on-mobile .govuk-breadcrumbs__list-item {
+
+  margin-left: 0;
+  padding-left: 0;
+  margin-right: 20px;
+  @include chevron($direction: "right", $color: govuk-colour("dark-grey"));
+
+  &:before,
+  &:last-of-type:after {
+    display: none;
+  }
+
+  @include govuk-media-query($until: tablet) {
+    display: none;
+    margin-bottom: 0;
+    padding: 14px 0;
+
+    &:last-of-type {
+      display: inline-block;
+      padding-left: 0;
+      margin-right: 0;
+      max-width: calc(80% - 14px);
+    }
+
+    &:first-of-type {
+      display: inline-block;
+      padding-left: 0;
+    }
+
+    &.gem-c-breadcrumbs--parent {
```

Comment:
Where is this class used? I can't see it in the review app.

Reply:
This was left from the original code prior to this work

Reply:
Can it be removed?

Reply:
Removed